<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VulnerabilityEditorWidget</class>
 <widget class="QWidget" name="VulnerabilityEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Vulnerabilities</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="pistolBox">
         <property name="text">
          <string>Pistol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="machineGunBox">
         <property name="text">
          <string>Machine gun</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="grenadeBox">
         <property name="text">
          <string>Grenade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="flamethrowerBox">
         <property name="text">
          <string>Flamethrower</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="axeBox">
         <property name="text">
          <string>Axe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bazookaBox">
         <property name="text">
          <string>Bazooka</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saberBox">
         <property name="text">
          <string>Saber</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="shieldBox">
         <property name="text">
          <string>Shield</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
